### ISSUE TITLE
CentOS7 - NFS mount doesn't survive a reboot

### ISSUE THREAD

**Issue by paulseward** [derdanne/puppet-nfs#63]
It appears that on CentOS7, the default setting for ::nfs::client_services_enable (false) leads to a scenario where rpcbind.service is running, but disabled - which means that systemd doesn't restart it after a reboot

The practical upshot is that after a reboot the NFS mount doesn't automatically re-establish (until after the next puppet run starts the service, which in my environment might be as much as an hour later)

Additionally, I now get the following on every puppet run:
```
Notice: /Stage[main]/Nfs::Client::Service/Service[rpcbind.service]/enable: enable changed 'true' to 'false'
```

This is the inverse of the behaviour described in #55!

I've been able to work around this by overriding the variable and setting it to `true` in the profile I'm using, and that seems to have no ill effect.  Puppet runs are now clean, with no idempotency problems.

I'm running the agent from PE2016-5 (4.8.1) - so I suspect the flip in behaviour for handling 'indirect' systemd services is possibly related to the puppet agent version.

I'm not sure what the best way to approach fixing this in the module is, as it seems that setting client_services_enable to true on CentOS7 causes idempotency problems for some people, but setting it to false appears to cause remounting at boot to fail.

**Comment by derdanne** [comment]
Hi thanks for the report!

I think we can reenable the service. This seemed to be fixed with PUP 4.9.0:

https://tickets.puppetlabs.com/browse/PUP-6759

**Comment by derdanne** [comment]
Reenabled in master branch
